capstone_project/src/main/webapp/scripts/maps_script.js

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Toggle to hide news feed if it is already showing when the
+ * button is pressed again.
+ * @param {string} schoolName
+ * @return {boolean}
+ */
+function toggleToHideFeed(schoolName) {
+  const x = document.getElementById(schoolName);
+  if (x.innerHTML !== '') {
+    x.innerHTML = '';
+    return true;
+  }
+  return false;
+}
+
+/** Generates a hard-coded feed of news articles for UCI. */
+function showUCINews() { // eslint-disable-line no-unused-vars
```

Comment:
Added a config comment to ignore the "defined but never used" rule, since my html file calls this function, not the js file. I did the same for the showUWNews() and showUWBNews() functions.